jax/_src/test_util.py

```diff
@@ -845,6 +845,18 @@ def _CheckAgainstNumpy(self, numpy_reference_op, lax_op, args_maker,
                         atol=atol or tol, rtol=rtol or tol,
                         canonicalize_dtypes=canonicalize_dtypes)
 
+_CPP_JITTER = functools.partial(api._jit, True)
```

Comment:
Perhaps JIT_IMPLEMENTATION instead of JITTER?

Same everywhere.

Reply:
Will do